brave/brave-core
# Refactor Brave Ads mojom to use mojo_base.mojom.Value/data structures instead of JSON

Resolves https://github.com/brave/brave-browser/issues/22506

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm notification ads are served
- Confirm new tab page ads are served
- Confirm inline content ads are served
- Confirm estimated earnings, next payment date and ads received are shown at brave://rewards
- Confirm diagnostics are shown at brave://rewards-internals > Ad diagnostics
